src/components/formio-wrapper.js

```diff
@@ -828,7 +834,8 @@ export class FormioWrapper {
     const emailButton = this.config.form.queryElement.querySelector(
       '[name="data[emailButton]"',
     );
-    if (this.wizard.data.sendEmail === 'user') {
+    const wizardData = JSON.parse(JSON.stringify(this.wizard.data));
```

Comment:
unsure if wizard data has the full data, have we tried wizard.submission.data ?

Reply:
To my knowledge, we have not used wizard.submission.data in the code. so wizard.data should be containing all data. @asifamingov  could you please confirm?

Reply:
Yes it should contain all data.

Reply:
Thanks Travis